### DISCUSSION TITLE
Can we use extra data in line series?

### DISCUSSION THREAD

**Discussion by Gilson401** [tradingview/lightweight-charts#1097] [Q&A]
In lineChart, series.setData() accepts an array of objects containing {     time: 'some_time_value',     value: 'some_price_value'  }.
I wonder add others key-values in this object to get them at crosshairMove and display more data in tooltip then only current point price and time. Is it possible?
Example code:

`  function  updateChartData() {
      const btcValues = []
      const assetPrices = []
      const termoValues = []

      this.termoBtcData.forEach((p) => {
        btcValues.push({
          time: p.rptDate,
          value: p.btcVar,
          btc_financial: p.btc_financial,
        })
        termoValues.push({
          time: p.rptDate,
          value: p.termoVar,
          termo_financial: p.termo_financial,
        })
        assetPrices.push({ time: p.rptDate, value: p.priceVar, close: p.close })
      })

      this.btcSeries.setData(btcValues)
      this.termoSeries.setData(termoValues)
      this.priceSeries.setData(assetPrices)

      this.bgtChart.priceScale().applyOptions({ autoScale: true })

      this.bgtChart.timeScale().fitContent()
    },
`

**Comment by edew** [comment]
Hi @Gilson401 

I the library won't return this extra data for you as it only cares about prices and times.

But seeing as you provide the data you could lookup the data you need in the event handler.

Something like this https://codesandbox.io/s/nostalgic-violet-3st0oq?file=/index.html
